.github/workflows/php-changes-detection.yml

```
@@ -21,8 +21,8 @@ jobs:
               with:
                   files: |
                       *.{php}
```

Wouldn't changing this as such and removing the other lines have the same effect?

```suggestion
                      **.{php}
```
`*` matches one or more characters except `/`, where as `**` matches any characters ([documentation](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)).

The goal of this is to run anytime a PHP file is edited. So regardless of the path to the `.php` file, this should run.

---

Thanks for the review, updated 👍